manifests/resource/upstream.pp

```diff
@@ -10,6 +10,7 @@
 #   [*upstream_cfg_prepend*]  - It expects a hash with custom directives to put before anything else inside upstream
 #   [*upstream_fail_timeout*] - Set the fail_timeout for the upstream. Default is 10 seconds - As that is what Nginx does normally.
 #   [*upstream_max_fails*]    - Set the max_fails for the upstream. Default is to use nginx default value which is 1.
+#   [*upstream_weight*]       - Set the weight for the upstream. Default is to use nginx default value which is 1.
```

Comment:
reading the erb template it seems to me the default is not to set the param and use whatever default nginx has. 

Reading this line I'd expect that it writes the param with value 1 in config if I don't specify anything.